techeria_controller/techeria_app/models.py

```diff
@@ -38,15 +38,19 @@ class Meta:
 class SellerModel(models.Model):
     first_name = models.CharField(max_length=255)
     last_name = models.CharField(max_length=255)
-    user_name = models.CharField(max_length=255)
+    username = models.CharField(max_length=255)
     date_of_birth = models.DateField()
     email = models.EmailField()
-    mobile_number = models.IntegerField(max_length=10)
```

Comment:
Here also we can add phonenumber_field library

Reply:
got it. I will add the mobile number field for the  mobile_number